docs/api/modules/main/exports/schema.md

```diff
@@ -1055,6 +1055,52 @@ schema.queryType({
  */
 ```
 
+### `importType`
+
+##### Signature
+
+```ts
+schema.importType(scalarType: GraphQLScalarType, methodName?: string): GraphQLScalarType
+schema.importType(type: GraphQLNamedType): GraphQLNamedType
+```
+
+`schema.importType` is useful for adding existing GraphQL.js types into your Nexus schema.
+c
+[Check out this repository](https://github.com/Urigo/graphql-scalars) for a handful list of useful scalar types that might be useful to your GraphQL API.
+
+When passing a `GraphQLScalarType`, you can additionally pass a `methodName` as a second parameter, which will augment the `t` parameter of your definition builder with a convenient method to create a field of the associated type.
+
+##### Example: Adding a date scalar type
+
+```ts
+// graphql.ts
+import { schema } from 'nexus'
+import { GraphQLDate } from 'graphql-iso-date'
+     
+schema.importType(GraphQLDate, 'date')
+     
+schema.objectType({
+  name: 'SomeObject',
+  definition(t) {
+    t.date('createdAt') // t.date() is now available (with types!) thanks to `importType`
+  },
+})
+```
+
+`schema.importType` can also be used to add types from an existing GraphQL schema into your Nexus schema. This is useful to incrementally adopt Nexus if you already have a GraphQL schema built with a different technology than Nexus.
```

Comment:
I get that this is co-located with the example but overall the approach so far has been front loaded description following by examples. Feel strongly about this one?

Reply:
Well, so far we also didn't have apis that had overload, and I felt like explaining the two overload in a same paragraph felt weird.

So right now we have on top-level general explanation

> `schema.importType` is useful for adding existing GraphQL.js types into your Nexus schema.

> [Check out this repository](https://github.com/Urigo/graphql-scalars) for a > handful list of useful scalar types that might be useful to your GraphQL API.

then the examples, which have some explanations and which are used to more specifically explain each overload

I also thought about having two `Signature` block but I ended up going with the following pattern to keep it simple